documentation/modules/rn-2.5.adoc

```diff
@@ -26,29 +26,30 @@ In this version of {project-short}, migration using OpenStack source providers g
 
 .Disabling FIPS
 
-EMS enforcement is disabled for migrations with VMware vSphere source providers to enable migrations from versions of vSphere that are supported by {project-short} but do not comply with the 2023 FIPS requirements.
+{project-short} enables migrations from vSphere source providers by not enforcing Enterprise Master Secret (EMS). This enables migrating from all vSphere versions that {project-short} supports, including migrations that do not meet 2023 FIPS requirements.
 
 .Integration of the create and update provider user interface
 
-The user interface of create and update providers now aligns with the look and feel of the {ocp} web console and displays up-to-date data.
+The user interface of the create and update providers now aligns with the look and feel of the {ocp} web console and displays up-to-date data.
 
 .Standalone UI
 
-The old UI of MTV 2.3 cannot be enabled by setting `feature_ui: true` in ForkliftController anymore.
+The old UI of {project-short} 2.3 cannot be enabled by setting `feature_ui: true` in ForkliftController anymore.
 
-.Errors logged in populator pods are improved
+.Support deployment on {ocp-name} 4.15
+
+{project-short} 2.5.6 can be deployed on {ocp-name} 4.15 clusters, with the resolution of two issues: 
 
-In previous releases of {project-short} {project-version}, populator pods were always restarted on failure. This made it difficult to gather the logs from the failed pods. In {project-short} 2.5.3, the number of restarts of populator pods is limited to three times. On the third and final time, the populator pod remains in the fail status and its logs can then be easily gathered by must-gather and by forklift-controller to know this step has failed. link:https://issues.redhat.com/browse/MTV-818[(MTV-818)]
+* The Forklift operator includes a fix for the link:https://issues.redhat.com/browse/OCPBUGS-30095[Forklift operator failing on {ocp-name} 4.15]. 
```

Comment:
just noticed that the security level of https://issues.redhat.com/browse/OCPBUGS-30095 has changed and it's longer public..
we had back and forth with the description of this item, but at this point I think we should drop the details of the "two issues", and just say that MTV 2.5.6 can be deployed on OpenShift 4.15 cluster